.github/workflows/release.yml

```diff
@@ -149,3 +149,10 @@ jobs:
         foreach ($example in $examples) { dotnet build $example.FullName }
       if: runner.os == 'Windows'
       shell: powershell
+    - name: 🔗 Install Templates
+      run: |
+        dotnet new install $env:USERPROFILE/.nuget/packages/icerpc.templates
+      shell: bash
```

Comment:
This is not quite correct. `$env:USERPROFILE/` is a powershell expression and we are setting shell bash. Unlike `CI` workflow this runs in Windows, macOS and Linux. I think the simpler is to install the step that install NuGet packages to also install templates, something like:

```
--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -116,16 +116,18 @@ jobs:
       with:
         name: packages
         path: packages
-    - name: Install packages to local NuGet repository
+    - name: Install NuGet packages and dotnet new templates
       working-directory: packages
       run: |
         dotnet nuget push *.nupkg --source ~/.nuget/packages
+        dotnet new install IceRpc.Templates*.nupkg
       if: runner.os == 'macOS' || runner.os == 'Linux'
       shell: bash
-    - name: Install packages to local NuGet repository
+    - name: Install NuGet packages and dotnet new templates
       working-directory: packages
       run: |
         dotnet nuget push *.nupkg --source $env:USERPROFILE/.nuget/packages
+        dotnet new install IceRpc.Templates*.nupkg
       if: runner.os == 'Windows'
       shell: powershell
```

Reply:
Oh okay wasn't sure when I saw it if it was some special action syntax I wasn't aware of. Will fix.